[2022-04-24 23:26:53,061] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: local_2_gcs_2_bq_dag.delete_people_dataset_task manual__2022-04-24T22:27:24.245486+00:00 [queued]>
[2022-04-24 23:26:53,077] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: local_2_gcs_2_bq_dag.delete_people_dataset_task manual__2022-04-24T22:27:24.245486+00:00 [queued]>
[2022-04-24 23:26:53,082] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-24 23:26:53,085] {taskinstance.py:1239} INFO - Starting attempt 3 of 4
[2022-04-24 23:26:53,086] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-24 23:26:53,108] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): delete_people_dataset_task> on 2022-04-24 22:27:24.245486+00:00
[2022-04-24 23:26:53,142] {standard_task_runner.py:52} INFO - Started process 2994 to run task
[2022-04-24 23:26:53,152] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'local_2_gcs_2_bq_dag', 'delete_people_dataset_task', 'manual__2022-04-24T22:27:24.245486+00:00', '--job-id', '123', '--raw', '--subdir', 'DAGS_FOLDER/download_2_gcs_dag_V3.py', '--cfg-path', '/tmp/tmp4m00xwvt', '--error-file', '/tmp/tmp2aaybvoq']
[2022-04-24 23:26:53,155] {standard_task_runner.py:77} INFO - Job 123: Subtask delete_people_dataset_task
[2022-04-24 23:26:53,262] {logging_mixin.py:109} INFO - Running <TaskInstance: local_2_gcs_2_bq_dag.delete_people_dataset_task manual__2022-04-24T22:27:24.245486+00:00 [running]> on host 2bdc5459fc64
[2022-04-24 23:26:53,378] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=local_2_gcs_2_bq_dag
AIRFLOW_CTX_TASK_ID=delete_people_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-24T22:27:24.245486+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-04-24T22:27:24.245486+00:00
[2022-04-24 23:26:53,382] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-04-24 23:26:53,385] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'rm /opt/***/u6pd-qa9d.csv /opt/***/u6pd-qa9d.parquet']
[2022-04-24 23:26:53,416] {subprocess.py:85} INFO - Output:
[2022-04-24 23:26:53,651] {subprocess.py:93} INFO - Command exited with return code 0
[2022-04-24 23:26:53,692] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=local_2_gcs_2_bq_dag, task_id=delete_people_dataset_task, execution_date=20220424T222724, start_date=20220424T232653, end_date=20220424T232653
[2022-04-24 23:26:53,729] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-24 23:26:53,788] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
